.github/CODEOWNERS

```diff
@@ -0,0 +1,5 @@
+# Code owners file.
```

Comment:
I have déjà vu about this PR :)

Reply:
Yeah, we have to add it separately to each branch. Just initial overhead.